1.20.x Compatibility

DaringShepard opened the pull request:
Updates to make the plugin support 1.20.x. Seems to work fine on 1.20.4 at the moment. Some code has been used from https://github.com/pop4959/LWCX to help streamline the update process.

DaringShepard committed:
1.20.x Compatibility

Updates to make the plugin support 1.20.x. Seems to work fine on 1.20.4 at the moment. Some code has been used from https://github.com/pop4959/LWCX to help streamline the update process.

jascotty2 commented on core/src/main/java/com/griefcraft/util/matchers/WallMatcher.java:
These comments are unnecessary, it's already a comment block.

jascotty2 commented on core/src/main/java/com/griefcraft/model/LWCPlayer.java:
```diff
@@ -130,8 +132,14 @@ public String getName() {
+     */
+    @Override
+    public Spigot spigot() {
+        return null;
```
Does `player.spigot();` no longer work?

DaringShepard committed:
Tweaks

Re-enabled params for WallMatcher. For whatever reason I disabled them it seems to work fine now. Swapped over the return null in LWCPlayer for player.spigot. Plugin still compiles and runs seemingly without errors.
